wormhole-connect/src/config/utils.ts

```diff
@@ -144,7 +143,11 @@ export const mergeNttGroups = (
   return builtin;
 };
 
-export const validateDefaults = (defaults: BridgeDefaults | undefined) => {
+export const validateDefaults = (
+  defaults: BridgeDefaults,
+  chains: ChainsConfig,
+  tokens: TokensConfig,
```

Comment:
this solves a bug: when switching networks `config.chains` is still the chains from the previous network.